client.py

```diff
@@ -105,20 +91,30 @@ def main():
     if 'StartdLogging' in config_dict:
         config_startd_logging = config_dict['StartdLogging']
     else:
-        config_startdlogging = {}
+        config_startd_logging = {}
+
+    # Loading secrets.  Fail if permissions wrong.
+    if oct(stat.S_IMODE(os.stat(options.secrets).st_mode)) != "0600":
+        print('Set Permissions on {} to 600').format(options.secrets)
```

Comment:
`.format()` should be on the string, not on the print function

Reply:
Good catch.  I'll clean that up.  These sorts of things will be easier to test with build bot because we can give it multiple setups to exercise all of the code.